README.md

```diff
@@ -1,6 +1,8 @@
 Happa
 =====
 
+[![CircleCI](https://circleci.com/gh/giantswarm/happa/tree/master.svg?style=svg&circle-token=6e98ba111259986b590f228cd20e20fcea3dd2e5)](https://circleci.com/gh/giantswarm/happa/tree/master)
```

Comment:
```suggestion
[![CircleCI](https://circleci.com/gh/giantswarm/happa/tree/master.svg?style=shield&circle-token=6e98ba111259986b590f228cd20e20fcea3dd2e5)](https://circleci.com/gh/giantswarm/happa/tree/master)
```

Reply:
out of curiosity, why?

Reply:
Because of consistency and beauty. We have it in all repos like that. The badge is nicer and aligns better with other shield style badges, which is the majority of badges out there AFAIK.